backend/notification/notif_controller.js

```diff
@@ -82,3 +83,19 @@ exports.updateAllOfflineNotifications = async (req, res) => {
     });
   }
 };
+exports.saveNewNotificationInteraction = async (req, res) => {
+  const userId = req.user.id;
+  const { category, notificationId } = req.body;
+  try {
+    const newInteraction = await rateLimiter.saveNotificationInteraction(
```

Comment:
More of a nitpick but I think it would make more sense to save the interaction data here than in the RateLimiter class, since interactions are directly related to notifs but not necessarily related to rate limiting

Reply:
Thanks. I have fixed it in https://github.com/izuchukwumba/peered/pull/20/commits/25122476dc6f2cf044ae1f51c3e04fba213dca0d